Notebooks/load_multiple_wells.ipynb

```diff
@@ -54,58 +54,226 @@
+      ],
       "text/plain": [
-       "<Figure size 1008x720 with 8 Axes>"
+       "<IPython.core.display.HTML object>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
+     "execution_count": 33,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "import matplotlib.pyplot as plt\n",
+    "import welly.quality as qty\n",
     "\n",
-    "fig, axs = plt.subplots(figsize=(14, 10), ncols=len(p))\n",
+    "trial= { 'All': [qty.no_similarities],\n",
+    "         'Each': [qty.no_monotonic, qty.not_empty, qty.no_gaps],\n",
+    "         'GR':[qty.all_positive,qty.all_between(0,250)],\n",
+    "         'DENS': [qty.all_between(1.5,3.5)],\n",
+    "         'DT': [qty.mean_between(40,250)],\n",
+    "         'DTS': [qty.mean_between(50,150)],\n",
+    "         'CAL':[qty.mean_between(2,24)],\n",
+    "         'THOR': [qty.mean_between(0,40)],\n",
+    "         'POTA': [qty.mean_between(0,100)],\n",
+    "         'URAN': [qty.mean_between(0,30)],\n",
+    "         'PE': [qty.mean_between(1.5,18)],\n",
+    "         'RES': [qty.mean_between(0.2,2000)]}\n",
+    "html = p.curve_table_html(keys=keys, alias=alias, tests=trial)\n",
+    "HTML(html)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# looks into root folder or subdirectories for LAS files\n",
+    "asps = []\n",
+    "for root, dirs, files in os.walk(r'C:\\Users\\jrogu\\Desktop\\Azzait\\las\\trialicillo'):\n",
```

Comment:
Same here it looks like this could be a useful function if we pass the root folder to it, maybe make a function in datadashviz

Reply:
made multiload.py to scan for las files